examples/LottieExtension.cpp

```diff
@@ -24,71 +24,105 @@
 /* Drawing Commands                                                     */
 /************************************************************************/
 
-static unique_ptr<tvg::LottieAnimation> animation;
-static Elm_Transit *transit;
-static bool updated = false;
+static unique_ptr<tvg::LottieAnimation> slotAnimation;
+static unique_ptr<tvg::LottieAnimation> markerAnimation;
+static Elm_Transit *slotTransit;
+static Elm_Transit *markerTransit;
+static int updated = 0;
```

Comment:
`LottieExtension` will have been keeping Lottie specified features here.
These definitions seem heavy, it will definitely need duplicated code.

Can we rewrite the multi animation examples under the struct or array?
Something like this:
```cpp
struct LottieExample
{
    enum Type : uint8_t { Slot = 0, Marker }
    unique_ptr<tvg::LottieAnimation> animation;
    Elm_Transit transition;
}

static std::vector<LottieExample> examples;
```